src/components/SourcesSimpleView/formatters.js

```diff
@@ -59,11 +67,38 @@ export const nameFormatter = (name, source, { sourceTypes }) => (
     </TextContent>
 );
 
-export const defaultFormatter = (value) => `undefined formatter for: ${value}`;
+export const defaultFormatter = (name) => (value) => `undefined ${name} formatter of value: ${value}`;
+
+export const importedFormatter = (value) => {
+    if (!value) {
+        return null;
+    }
+
+    const text = importsTexts(value);
+
+    if (text) {
+        return (<Tooltip content={text}>
+            <Badge isRead>
```

Comment:
Maybe add `cursor: help` so the user will know that eventually a tool tip will show.

Reply:
great idea :exclamation: